test/conftest.py

```diff
@@ -246,6 +247,27 @@ def planets_zebeth_game_description() -> GameDescription:
     return default_database.game_description_for(RandovaniaGame.METROID_PLANETS_ZEBETH)
 
 
+@pytest.fixture(scope="session")
+def prime_hunters_game_description() -> GameDescription:
```

Comment:
Let's keep these fixtures to `test/games/prime_hunters/conftest.py`

Reply:
All of them? I see other games have theirs in here.

Reply:
Some are also just repeating a bad pattern, but blank /Echoes are used to treat clue functionality

Reply:
Then I don't know what I'm being asked to do. Unless I was supposed to move all of them over, which I thought was giving me errors which is why I didnt.